spec/helpers/perf/drivers/terraform.lua

```diff
@@ -166,14 +166,31 @@ end
 
 function _M:start_upstream(conf)
   conf = conf or ""
-  conf = ngx.encode_base64(([[server {
-              listen %d;
-              access_log off;
-              location =/health {
-                return 200;
-              }
-              %s
-            }]]):format(UPSTREAM_PORT, conf)):gsub("\n", "")
+  conf = ngx.encode_base64(([[
+  worker_processes auto;
+  worker_cpu_affinity auto;
+  error_log /var/log/nginx/error.log;
+  pid /run/nginx.pid;
+  worker_rlimit_nofile 20480;
```

Comment:
Out of curiosity: are these hardcoded values determined empirically? (Understandably, they have to be set to a "high enough" value to not limit the benchmarks, but, in this case, how did you get to those?)

Reply:
I duplicated it from a performance test done by a third party with us and a competitor. I didn't really validate it with comparasion but from a first view I feel it's sane.